ModernWpf.Controls/NavigationView/NavigationView.cs

```diff
@@ -546,8 +546,8 @@ public override void OnApplyTemplate()
             if (GetTemplateChild(c_topNavOverflowButton) is Button topNavOverflowButton)
             {
                 m_topNavOverflowButton = topNavOverflowButton;
-                AutomationProperties.SetName(topNavOverflowButton, Strings.NavigationOverflowButtonName);
-                topNavOverflowButton.Content = Strings.NavigationOverflowButtonText;
+                AutomationProperties.SetName(topNavOverflowButton, ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_NavigationOverflowButtonName));
```

Comment:
nit: if we add `using static ModernWpf.ResourceAccessor` then the second `ResourceAccessor` can be omitted. Actually, the first `ResourceAccessor` can be omitted too, but it might be a good idea to keep it to be consistent with WinUI.

Reply:
Yep! Would do that.